updated the ARIA tag as per casing being used

ctrlaltvikas opened the pull request.

mikekreuzer committed:
Centre container on IE

- add width: 32rem;
- alpha order attributes

ctrlaltvikas committed:
Added open graph meta tags

pubudu-ranasinghe committed:
Add live server for development

ctrlaltvikas committed:
removed og:image tag

ctrlaltvikas committed:
updated the ARIA tag as per casing being used

tl0 commented:
Should it still be without hyphen though? [w3](https://www.w3.org/TR/wai-aria-1.1/#contentinfo), also Google Chrome's Lighthouse gives following warning if kebab-case is being used ![alt text](https://i.imgur.com/7Onl7C2.png)

Commit:
Fix unnecessary newlines in responsive footer

Commit:
Eliminate chance of zero horizontal margins

Horizontal margins being the spacing between the body content
and the viewport edges. This previously occurred on some screen sizes.

ctrlaltvikas commented:
I think it will be fine. Your call.



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality3&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality3&>

ctrlaltvikas committed:
Adding sitemap.xml file

ankit-m committed:
Add link to GitHub contributors

bryanwtan committed:
split css statements into lines

MattPetitt committed:
Updating serve to most recent version (#68)

crisdosaygo committed:
Gradient fixes (#65)

* Fixed issue with gradient height being less than page height
The issue arises in some situations (such as when scrolling on a small screen)
as raised by @MehediH in https://github.com/Microsoft/join-dev-design/pull/46#issuecomment-406570731

* Removed display: flex I mistakenly committed

dvdsgl committed:
Merge pull request #59 from bryanwtan/improve-devtools-css-readability

Split devtools css statements into lines

d4rekanguok committed:
js to toggle theme

dvdsgl committed:
Merge pull request #58 from awfulaxolotl/spacing-fixes

Minor spacing fixes

dvdsgl committed:
Merge pull request #64 from ankit-m/add-contributor-link

Add link to GitHub contributors

pubudu-ranasinghe committed:
use live-server only

Commit:
update transform property to use GPU for smoother animation

d4rekanguok committed:
basic functionality

Commit:
make font sizing relative and consistent REM units

d4rekanguok committed:
stying

Commit:
adding valid aria roles to HTML

ctrlaltvikas committed:
updated og:image tag again

d4rekanguok committed:
fade in

ctrlaltvikas committed:
added image for og:image tag

d4rekanguok committed:
more styling

d4rekanguok committed:
put font back

ctrlaltvikas committed:
changed the url

mkivikoski committed:
footer: change role from 'footer' to 'contentinfo' as 'footer' is an invalid value'

dvdsgl committed:
Merge pull request #75 from mkivikoski/bug/footer-role-is-invalid

footer: change role from 'footer' to 'contentinfo' as 'footer' is an invalid value

dvdsgl committed:
Restore Figma shout-out

dvdsgl committed:
Merge branch 'master' into theme

dvdsgl committed:
Merge pull request #51 from pubudu-ranasinghe/master

Add live server for development

d4rekanguok committed:
add missing closing div

dvdsgl committed:
Merge pull request #39 from mikekreuzer/master

Centre container on IE

mkivikoski committed:
content: add links to microsoft sites for diversity and openess resources

mkivikoski committed:
content: add link to MS design for 'delightful tools'

d4rekanguok committed:
reformat index.html

isthisnagee committed:
Add editor config

dvdsgl committed:
Merge pull request #76 from mkivikoski/feature/link-to-diversity-and-openess-screens

content: add links to microsoft sites for diversity and openess resources

d4rekanguok committed:
reformat index.html

d4rekanguok committed:
reformat index.html final

dvdsgl committed:
Smaller margins on narrow screens

d4rekanguok committed:
move theme selector to below the last paragraph

d4rekanguok committed:
reformat

dvdsgl committed:
Merge pull request #73 from d4rekanguok/theme

Add themes

dvdsgl committed:
Merge pull request #77 from isthisnagee/master

Add editor config

dvdsgl committed:
Merge pull request #47 from vikaskyadav/patch-1

Added open graph meta tags

dvdsgl committed:
Fix theme script path

dvdsgl committed:
Merge pull request #78 from danielroot/improvement

Some updates for font-sizing, accessibility, and animation performance

Commit:
MSFT logo on console using gradients

dvdsgl committed:
Merge pull request #79 from sangeeth96/nerd-log-fix

MSFT logo on console using gradients

pubudu-ranasinghe committed:
show pointer on hover over theme buttons for better ux

dvdsgl committed:
Merge pull request #82 from pubudu-ranasinghe/master

show pointer on hover over theme buttons for better ux

dvdsgl committed:
Allow browser to open

Commit:
Added prettier and pre-commit hooks

Commit:
prettier formatting fixes

dvdsgl committed:
Merge pull request #84 from sangeeth96/setup-prettier

Prettier code formatting added... need more tooling?

ctrlaltvikas committed:
Merge branch 'master' into patch-2
